src/StatusQ/Controls/StatusRoundButton.qml

```diff
@@ -86,9 +86,9 @@ Rectangle {
         readonly property color iconColor: statusRoundButton.enabled ? statusRoundButton.icon.color : statusRoundButton.icon.disabledColor
     }
 
-    implicitWidth: 44
-    implicitHeight: 44
-    radius: width / 2;
+    implicitWidth: 45
```

Comment:
logically 22/44 should do the job however I see in the screenshot that in this case with the arrow icon 22/45 looks slightly better. did you try with other icons?